src/ast/irbuilderbpf.cpp

```diff
@@ -427,14 +436,14 @@ Value *IRBuilderBPF::CreateStrcmp(Value* val1, Value* val2, bool inverse) {
   */
   Function *parent = GetInsertBlock()->getParent();
   BasicBlock *str_ne = BasicBlock::Create(module_.getContext(), "strcmp.false", parent);
-  AllocaInst *store = CreateAllocaBPF(getInt8Ty(), "strcmp.result");
+  AllocaInst *store = CreateAllocaBPF(getInt64Ty(), "strcmp.result");
```

Comment:
why this change?

Reply:
This was added due to [this comment](http://www.github.com/iovisor/bpftrace/pull/778#issuecomment-515167110), essentially doing a simple compare in an if statement was failing because of a type mismatch. I'm not sure why this is causing the map lookup code to be removed (reverting this does fix that), I'll try to figure out why that's happening...